pkg/rpcclient/wsclient.go

```diff
@@ -520,6 +566,14 @@ readloop:
 				ntf.Value = new(state.AppExecResult)
 			case neorpc.NotaryRequestEventID:
 				ntf.Value = new(result.NotaryRequestEvent)
+			case neorpc.HeaderEventID:
+				sr, err := c.stateRootInHeader()
+				if err != nil {
+					// Client is not initialized.
+					connCloseErr = fmt.Errorf("failed to fetch StateRootInHeader: %w", err)
+					break readloop
+				}
+				ntf.Value = block.New(sr).Header
```

Comment:
Are you sure that this code works properly? Don't you need to return a pointer to header?